src/kw_config_loader.sh

```diff
@@ -27,14 +28,41 @@ function show_variables()
     has_local_config_path="No"
   fi
 
+  if [[ "$1" == 'TEST_MODE' ]]; then
+    test_mode='1'
+  fi
+
+  local -A descriptions=(
+    [arch]='Target arch'
+    [mount_point]='Mount point'
+    [virtualizer]='Virtualization tool'
+    [qemu_hw_options]='QEMU options'
+    [qemu_net_options]='QEMU Net options'
+    [ssh_ip]='Default ssh ip'
+    [ssh_port]='Default ssh port'
+    [kernel_img_name]='Kernel image name'
+    [cross_compile]='Cross-compile name'
+    [menu_config]='Default kernel menu config'
+    [qemu_path_image]='Default virtualization tool'
```

Comment:
That's the QEMU image path, not a "virtualization tool"

Reply:
Oops, that was a copy-paste I forgot to alter later...